### PR TITLE
DS-746 | @ericbakenhus | Additional copy changes

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Copy changes
- A11y changes

# Review By (Date)
- ASAP

# Criticality
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code
2. Navigate to [/citizenship/](https://deploy-preview-422--adapt-giving.netlify.app/citizenship/)
3. Verify it looks fine

# Associated Issues and/or People
- DS-746

### CODE REVIEW

File: src/pages/citizenship.js [hunk 1]
```diff
@@ -30,50 +30,56 @@ const ForeignSourceReporting = ({ data }) => {
...
-          reporting thresholds. Thank you for providing your information below.
-        </p>
-        <p>
-          You can find additional details about this policy in section V of
-          Stanford’s Gift Policy website:{' '}
-          <SbLink
-            linkType="url"
-            link={{
-              url: 'https://giving.stanford.edu/gift-policy',
+      <main id="main-content">
```
Comment (ericbakenhus) [inline src/pages/citizenship.js:33]: Added `main`. Skip nav expects it to exist.

File: src/pages/citizenship.js [hunk 2]
```diff
@@ -30,50 +30,56 @@ const ForeignSourceReporting = ({ data }) => {
...
+            <SbLink
+              linkType="url"
+              link={{
+                url: 'https://giving.stanford.edu/gift-policy',
+              }}
+            >
+              Gift Policy website
+            </SbLink>
+            .
+          </p>
```
Comment (ericbakenhus) [inline src/pages/citizenship.js:66]: Copy changes.

File: src/pages/citizenship.js [hunk 3]
```diff
@@ -30,50 +30,56 @@ const ForeignSourceReporting = ({ data }) => {
...
+        </CenteredContainer>
+        <div>
+          <iframe
+            title="Citizenship form for foreign source reporting"
```
Comment (ericbakenhus) [inline src/pages/citizenship.js:70]: Added title to iframe so we meet 4.1.2.

File: src/pages/citizenship.js [hunk 4]
```diff
@@ -30,50 +30,56 @@ const ForeignSourceReporting = ({ data }) => {
...
-        </p>
-        <p>
-          You can find additional details about this policy in section V of
-          Stanford’s Gift Policy website:{' '}
-          <SbLink
-            linkType="url"
-            link={{
-              url: 'https://giving.stanford.edu/gift-policy',
+      <main id="main-content">
+        <CenteredContainer classes="su-max-w-800">
```
Comment (ericbakenhus) [inline src/pages/citizenship.js:34]: I tossed a max width on the text to get closer to meeting 1.4.8. It's technically AAA, but it seemed like a good idea.

File: src/pages/citizenship.js [hunk 5]
```diff
@@ -30,50 +30,56 @@ const ForeignSourceReporting = ({ data }) => {
...
+            received from a donor who is associated with one of certain specific
+            countries (the list, which may change over time, currently includes
+            China (while excluding Hong Kong), Russia, Iran, and North Korea),
+            when that donor makes gifts and other payments to Stanford having an
+            aggregate value of $50,000 or more during a reporting period that
+            runs from July 1 to June 30 of the following year. Because your
+            generous gift(s) meet one or both of these reporting thresholds, we
+            ask that you please provide the information requested below. Thank
+            you.
+          </p>
```
Comment (ericbakenhus) [inline src/pages/citizenship.js:53]: Copy changes.

**Review by sherakama — APPROVED** [review]
Ty

File: src/pages/citizenship.js [hunk 6]
```diff
@@ -30,50 +30,56 @@ const ForeignSourceReporting = ({ data }) => {
...
+            You can find additional details about this policy in section V of
+            Stanford’s{' '}
+            <SbLink
+              linkType="url"
+              link={{
+                url: 'https://giving.stanford.edu/gift-policies/',
+              }}
+            >
+              Gift Policy website
+            </SbLink>
```
Comment (ericbakenhus) [inline src/pages/citizenship.js:64]: Fix link and make text descriptive (2.4.4).

**Pull request merged**